src/parser.ts

```diff
@@ -0,0 +1,15 @@
+export function parse(text : string, pos : number) {
+    let moduleName;
```

Comment:
Better to initialize it with `null`, as you are checking it at https://github.com/GodotExplorer/godot-tools/pull/110/files#diff-7edc58f3ac9d8d26d67594236c11bee6R46